docs/install_guides/configurations-for-intel-gna.md

```diff
@@ -16,7 +16,29 @@ Intel® GNA hardware requires a driver to be installed on the system.
 
 ## Linux
 
-[Download Intel® GNA driver for Ubuntu Linux 18.04.3 LTS (with HWE Kernel version 5.4+)](https://storage.openvinotoolkit.org/drivers/gna/)
+### Prerequisites
```

Comment:
how about just referencing README file from within linux driver package, without copying instruction here?
this way we have only one place to update in case of changes of module compilation/load procedure.

Reply:
If the readme file is put somewhere on Github, we can add a link to it. But from what you described, it's within the package, right? Then I think it's better to have the steps explicitly here, as users can get an idea of what need to be done.

Reply:
ok, then description is ok